checker/check-request-discriminator-updated.go

```diff
@@ -0,0 +1,148 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+func RequestDiscriminatorUpdatedCheck(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config Config) Changes {
```

Comment:
Add checker to defaultChecks

Reply:
good catch, thank you.
Updated defaultChecks